### DISCUSSION TITLE
【灌水】黑幕！给铁道花了这么多钱就给一张票？！

### DISCUSSION THREAD

**Discussion by BJ-cmyk** [share121/inter-knot#5371] [灌水]
![1727164512617](https://github.com/user-attachments/assets/c4868106-8c5d-47ae-bf25-cb2e8bdfa992)

**Comment by aino2333** [comment]
你这列车能在宇宙里开吗🤔

**Comment by ZionHuang28** [comment]
198和160一张不是差不多吗

**Comment by kinich1111** [comment]
回家的票一秒钟就没了，各位有什么头绪吗QAQ
